src/main/java/com/example/HomeWorkAug080823/model/Faculty.java

```diff
@@ -0,0 +1,71 @@
+package com.example.HomeWorkAug080823.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+
+import java.util.Objects;
+
+@Entity
+public class Faculty {
+    @Id
+    @GeneratedValue
```

Comment:
```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Лучше определить стратегию.
Аналогично у студентов.